components/infobox/wikis/freefire/infobox_person_custom.lua

```diff
@@ -0,0 +1,177 @@
+---
+-- @Liquipedia
+-- wiki=freefire
+-- page=Module:Infobox/Person/Player/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Player = require('Module:Infobox/Person')
+local String = require('Module:StringUtils')
+local Class = require('Module:Class')
+local Page = require('Module:Page')
+local TeamHistoryAuto = require('Module:TeamHistoryAuto')
+local Variables = require('Module:Variables')
+local Template = require('Module:Template')
+
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local Title = require('Module:Infobox/Widget/Title')
+local Center = require('Module:Infobox/Widget/Center')
+
+local _EMPTY_AUTO_HISTORY = '<table style="width:100%;text-align:left"></table>'
+
+local _BANNED = {['banned'] = {category = 'Banned players', variable = 'Banned', isplayer = true},}
```

Comment:
1) doesn't make much sense, you would set a table into statuscontents, while it expects a string
2) doesn't match the current wikicode:
```
{{#switch:{{{banned|}}}
   | vac = [[Banned players#Currently Banned from All PUBG Competitive events|VAC Banned]]
   | compban = [[Banned players#Currently Banned from All PUBG Competitive events|Competitive Banned]]
   | esl = [[Banned players#Partial_Bans|ESL Banned]]
   | faceit = [[Banned players#Partial_Bans|FACEIT Banned]]
   | starladder
   | sltv   = [[Banned players#Partial_Bans|SLTV Banned]]
   | wca   = [[Banned players#Partial_Bans|WCA Banned]]
   | power-lan = [[Banned players#partial_Bans|Power-LAN Banned]]
   |{{{banned}}} = [[Banned players|Multiple Bans]]}}}}<!--
-->{{#if:{{{banned2|}}}|<br>{{#switch:{{{banned2|}}}
   | vac = [[Banned players#Currently_Banned_from_Valve_Events|VAC Banned]]
   | compban = [[Banned players#Currently Banned from All PUBG Competitive events|Competitive Banned]]
   | esl = [[Banned players#Partial_Bans|ESL Banned]]
   | faceit = [[Banned players#Partial_Bans|FACEIT Banned]]
   | starladder
   | sltv   = [[Banned players#Partial_Bans|SLTV Banned]]
   | wca   = [[Banned players#Partial_Bans|WCA Banned]]
   | power-lan = [[Banned players#partial_Bans|Power-LAN Banned]]}}
```

Reply:
these all bans and nationalteams are from other wikis probably csgo and pubg.

Reply:
alright, so kicking the different ban reasons is intended
but that still leaves the issue that you push a table into statusContents where it expects a string

Reply:
Sorry, forgot to reply for the 1st question
Honestly idk why I have used that there at first I didn't added that, I added that from some other wiki yesterday because it was showing error (actually error because of I forgot to add `statusContents` to status) but I guess this can be removed 
(`	if String.isNotEmpty(_args.banned) then
		local banned = _BANNED[string.lower(_args.banned)]
		table.insert(statusContents, banned)
		end`)
because it already exists in Module:Infobox/Person as FF wiki only need to show status as banned(no extra prams)